drivers/peci/CMakeLists.txt

```diff
@@ -1,3 +1,5 @@
 # SPDX-License-Identifier: Apache-2.0
 
-zephyr_library()
```

Comment:
this looks strange. First it was removed and then added when it should look all green